docs/labwc-theme.5.scd

```diff
@@ -195,6 +195,52 @@ elements are not listed here, but are supported.
 	Height of boxes in workspace switcher in pixels. Setting to 0 disables
 	boxes. Default is 20.
 
+*snapping-preview.region.fill* [yes|no|auto]
+	Show filled rectangle as preview when window is snapped to a region.
+	Otherwise, outlined rectangle is shown instead.
+	When *auto* is specified, the style of a rectangle is automatically
+	determined depending on the current rendering backend which can be specified
+	with *WLR_RENDERER* environment variable.
+	Default is *auto*.
+
+*snapping-preview.edge.fill* [yes|no|auto]
+	Show filled rectangle as preview when window is snapped to an edge.
+	See *snapping-preview.region.fill* above for details.
+	Default is *auto*.
+
+*snapping-preview.region.bg.color*
+	Color of filled rectangle shown as preview when window is snapped to a
+	region.
+	Default is #8080b380.
```

Comment:
Default can be in the previous line, same for the following 3 entries. (I think it looks better when the the "Default is" line is shorter than the line above, in all other cases it should be in the previous line instead).